File: exercicios em Python/q28.py
import os
print("\n\nEste progrma determina se tres atomos sao isobaros, isotonos ou isotopos.")
a1 = int(input("\n\nDigite o numero de protons do primeiro atomo: "))
a2 = int(input("\nDigite o numero de protons do segundo atomo: "))
a3 = int(input("\nDigite o numero de protons do terceiro atomo: "))
	
b1 = int(input("\n\nDigite o numero de massa do primeiro atomo: "))
b2 = int(input("\nDigite o numero de massa do segundo atomo: "))
b3 = int(input("\nDigite o numero de massa do terceiro atomo: "))
	
c1 = int(input("\n\nDigite o numero de neutrons do primeiro atomo: "))
c2 = int(input("\nDigite o numero de neutrons do segundo atomo: "))
c3 = int(input("\nDigite o numero de neutrons do terceiro atomo: "))
	
if a1 == b1 & a1 == c1:
        print("\n\nOs tres atomos sao isotonos!")
        
elif a1 == b1 & b1 != c1:
        print("\n\nSomente o primeiro e segundo atomos sao isotanos!")
        
elif c1 == b1 & c1 != a1:
        print("\n\nSomente o segundo e o terceiro atomo sao isotanos!")

else:
        print("\n\nNenum dos atomos sao isotanos!")

if a2 == b2 & a2  == c2:
        print("\nOs tres atomos sao isobaros!")
        
elif a2 == b2 & a2 != c2:
        print("\nSomente o primeiro e segundo atomos sao isobaros!")
        
elif c2 == b2 & c2 != a2:
        print("\nSomente o segundo e o terceiro atomo sao isobaros!")

else:
        print("\n\nNenum dos atomos sao isobaros!")

if a3 == b3 & a3 == c3:
        print("\nOs tres atomos sao isotopos!")
        
elif a3 == b3 & a3 != c3:
        print("\nSomente o primeiro e segundo atomos sao isotopos!")

elif c3 == b3 & c3 != a3:
        print("\nSomente o segundo e o terceiro atomo sao isotopos!")

else:
        print("\nNenum dos atomos sao isotopos!")

print("\n\nObrigado por utilizar nosso programa.")
os.system("pause")

File: exercicios em C/q28.cpp
#include <stdio.h>

int main(void){
	int a1, b1, c1, a2, b2, c2, a3, b3, c3;
	printf("\n\nEste progrma determina se tres atomos sao isobaros, isotonos ou isotopos.");
	printf("\n\nDigite o numero de protons do primeiro atomo: ");
	scanf("%d", &a1);
	printf("\n\nDigite o numero de protons do segundo atomo: ");
	scanf("%d", &b1);
	printf("\n\nDigite o numero de protons do terceiro atomo: ");
	scanf("%d", &c1);
	
	printf("\n\nDigite o numero de massa do primeiro atomo: ");
	scanf("%d", &a2);
	printf("\n\nDigite o numero de massa do segundo atomo: ");
	scanf("%d", &b2);
	printf("\n\nDigite o numero de massa do terceiro atomo: ");
	scanf("%d", &c2);
	
	printf("\n\nDigite o numero de neutrons do primeiro atomo: ");
	scanf("%d", &a3);
	printf("\n\nDigite o numero de neutrons do segundo atomo: ");
	scanf("%d", &b3);
	printf("\n\nDigite o numero de neutrons do terceiro atomo: ");
	scanf("%d", &c3);
	
	if(a1 == b1 == c1){
		printf("\n\nOs tres atomos sao isotonos!");
	}
	else if (a1 == b1 & b1 != c1){
		printf("\n\nSomente o primeiro e segundo atomos sao isotanos!");
	}
	else if (a1 == c1 & c1 != b1){
		printf("\n\nSomente o primeiro e terceiro atomos sao isotanos!");
	}
	else if(c1 == b1 & b1 != a1){
		printf("\n\nSomente o segundo e o terceiro atomo sao isotanos!");
	}
	else {
		printf("\n\nNenum dos atomos sao isotanos!");
	}
		
	
	if(a2 == b2 == c2){
		printf("\n\nOs tres atomos sao isobaros!");
	}
	else if (a2 == b2 & b2 != c2){
		printf("\n\nSomente o primeiro e segundo atomos sao isobaros!");
	}
	else if (a2 == c2 & c2 != b2){
		printf("\n\nSomente o primeiro e terceiro atomos sao isobaros!");
	}
	else if(c2 == b2 & b2!= a2){
		printf("\n\nSomente o segundo e o terceiro atomo sao isobaros!");
	}
	else {
		printf("\n\nNenum dos atomos sao isobaros!");
	}
	
	if(a3 == b3 == c3){
		printf("\n\nOs tres atomos sao isotopos!");
	}
	else if (a3 == b3 & b3 != c3){
		printf("\n\nSomente o primeiro e segundo atomos sao isotopos!");
	}
	else if (a3 == c3 & c3 != b3){
		printf("\n\nSomente o primeiro e terceiro atomos sao isotopos!");
	}
	else if(c3 == b3 & b3 != a3){
		printf("\n\nSomente o segundo e o terceiro atomo sao isotopos!");
	}
	else {
		printf("\nNenum dos atomos sao isotopos!");
	}
	
	printf("\n\nObrigado por utilizar nosso programa.");
	return 0;
	
	
}
